### PR TITLE
Add tokenizer tool

### PR DESCRIPTION
**Pull request by abeaumont**
Depends on #1

### CODE REVIEW

**Review by smola — APPROVED** [review]

**Comment by abeaumont** [comment]
Rebased

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.5
```
Comment (juanjux) [inline Dockerfile:1]: I think we're using Alpine 3.6 now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+Once you have a server running, you can use the dummy tool, which
+should let you know if the connection with the server succeeded:
+
+`bblfsh-tools dummy path/to/source/code`
+
+If the server is in a different location, use the `address` parameter:
+
+`bblfsh-tools dummy --address location:port path/to/source/code`
+
+Once connection with the server is working fine, you can use any other
```
Comment (juanjux) [inline README.md:43]: "Once **the** connection"

**Review by erizocosmico — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (abeaumont) [reply Dockerfile:1]: updated

File: README.md (hunk 2 above)
Comment (abeaumont) [reply README.md:43]: fixed, thanks!

**Pull request merged**